editor/sidebar/post-status/index.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from 'i18n';
+import { Component } from 'element';
+import PanelBody from 'components/panel-body';
+import PanelBodyToggle from 'components/panel-body-toggle';
+import FormToggle from 'components/form-toggle';
+
+/**
+ * Internal Dependencies
+ */
+import './style.scss';
+
+class PostStatus extends Component {
+	constructor() {
+		super( ...arguments );
+		this.state = {
+			opened: true,
+		};
+		this.toggle = this.toggle.bind( this );
+	}
+
+	toggle( event ) {
+		event.preventDefault();
+		this.setState( {
+			opened: ! this.state.opened,
+		} );
+	}
+
+	render() {
+		return (
+			<PanelBody>
+				<PanelBodyToggle
```

Comment:
This makes me think we should add this behavior to the `PanelBody` instead, what do you think?

```js
<PanelBody toggle={ __( 'Status & Visibility' ) } initialOpen={ true }>
  <div className="editor-sidebar-post-status__row"></div>
</PanelBody>
```